decred/decred/dcr/txscript.py

```diff
@@ -2502,8 +2502,15 @@ def signP2PKHMsgTx(msgtx, prevOutputs, keysource, params):
 def paysHighFees(totalInput, tx):
     """
     paysHighFees checks whether the signed transaction pays insanely high fees.
-    Transactons are defined to have a high fee if they have pay a fee rate that
-    is 1000 time higher than the default fee.
+    Transactons are defined as having a high fee if they pay a fee rate that is
```

Comment:
~~Transacton~~ -> transaction, while you're here.

Reply:
such a hard word to spell right